go.mod

```diff
@@ -44,6 +44,7 @@ require (
 	google.golang.org/api v0.136.0
 	gopkg.in/DataDog/dd-trace-go.v1 v1.53.0
 	gopkg.in/yaml.v3 v3.0.1
+	gotest.tools/v3 v3.5.0
```

Comment:
This simplified writing tests a lot. It is like testify, but uses go-cmp under the hood. It's also made by the same [org](https://github.com/gotestyourself/) that make `gotestsum`, the tool we use to run go tests in CI.